deployer/scripts/validate.sh

```diff
@@ -33,29 +33,14 @@ function check_provided_certs() {
 }
 
 function check_service_accounts() {
+  common/groovy/validation.groovy check_edit_role "$project" || return
```

Comment:
We are still using alot of bash here.  I bet we could completely modify the script to use groovy instead of bash... with something like '#!/usr/bin/groovy`